data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v1.0.xml

```diff
@@ -105,5 +105,14 @@
         <addPrimaryKey columnNames="group_id, member_name"
                        constraintName="vnode_member_signature_pkey"
                        tableName="vnode_member_signature"/>
+
+        <createTable tableName="vnode_group_parameters">
+            <column name="epoch" type="INT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="parameters" type="VARBINARY(1048576)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
```

Comment:
`addPrimaryKey`

Reply:
There is a `primaryKey="true"` constraint on the epoch column. Do we still need to specify with `addPrimaryKey`?

Reply:
Oh, I didn't see that. I don't think we need it twice.